packages/inter-protocol/test/vaultFactory/test-storage.js

```diff
@@ -60,7 +60,6 @@ test('storage keys', async t => {
       'numActiveVaults',
       'numLiquidatingVaults',
       'numLiquidationsCompleted',
-      'retainedCollateral',
```

Comment:
I don't see why this goes away. (most likely I skimmed something too fast)

Reply:
It's an output of liquidation, which is temporarily missing.